devtools/conda-envs/test_env.yaml

```diff
@@ -24,7 +24,7 @@ dependencies:
   - numexpr
   - numpy >=1.14
   - openeye-toolkits
-  - openff-toolkit <0.11.0
+  - openff-toolkit >=0.11.0 # updated to latest
```

Comment:
We will need to make sure we also do this on conda-forge

Reply:
I think this works with 0.10.7 as well, but we probably want to support 0.11 or higher either way, because the protocols with the gufe objects will need that once we merge that into main. Thanks!